spec/controllers/api/v0/user_roles_controller_spec.rb

```diff
@@ -28,6 +29,28 @@
 
         expect(response.body).to eq(user_whose_delegate_status_changes.active_roles.to_json)
       end
+
+      it 'fetches list of banned competitos' do
+        get :index_for_group_type, params: { group_type: UserGroup.group_types[:banned_competitors] }
+
+        expect(response.body).to eq([banned_competitor].to_json)
+      end
+    end
+
+    context 'when user is logged in as a normal user' do
+      sign_in { FactoryBot.create(:user) }
+
+      it 'fetches list of roles of a user' do
+        get :index_for_user, params: { user_id: user_whose_delegate_status_changes.id }
+
+        expect(response.body).to eq(user_whose_delegate_status_changes.active_roles.to_json)
+      end
+
+      it 'fetches list of banned competitos' do
```

Comment:
The point of this test is that it does NOT fetch the list of banned competitors :P

Reply:
Thanks, changed. 👍